desidiff/nb/desidiff.ipynb

```diff
@@ -196,15 +211,66 @@
     "            plt.show()\n",
     "            count=count+1\n",
     "            \n",
+    "            #SkyPortal functionality begins:\n",
+    "            #Combine the wavelengths into one \n",
+    "            wavelengths = []\n",
+    "            for b in difflux.keys():\n",
+    "                wavelengths += difwave[b].tolist()\n",
+    "            \n",
+    "            flux = []\n",
+    "            #Combine the differenced flux into one \n",
+    "            for b in difflux.keys():\n",
+    "                flux += difflux[b].tolist()\n",
+    "            \n",
+    "            #Code to make wavelengths monotonic with corresponding changes in flux:\n",
+    "            wavelengths, flux = (list(t) for t in zip(*sorted(zip(wavelengths, flux))))\n",
```

Comment:
I am not familiar with this syntax, in particular, what happens in the sorting of a zip. We want to sort over wavelengths, the following is straightforward.

```
as = numpy.argsort(wavelengths)
wavelengths=wavelengths[as]
flux=flux[as]
```